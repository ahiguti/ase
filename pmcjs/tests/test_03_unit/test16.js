
var x = Perl.NewHash({ foo : 'fooval', bar : 'barval', あ : 'あval' });
for (var i in x) {
  print("k=[" + i + "] v=[" + x[i] + "]");
}
x = Perl.NewHash([ 12, 94, 117, "abc", "999" ]);
for (var i in x) {
  print("k=[" + i + "] v=[" + x[i] + "]");
}

