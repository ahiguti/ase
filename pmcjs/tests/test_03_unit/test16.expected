k=[あ] v=[あval]
k=[bar] v=[barval]
k=[foo] v=[fooval]
k=[4] v=[999]
k=[1] v=[94]
k=[3] v=[abc]
k=[0] v=[12]
k=[2] v=[117]
